github/Requester.pyi

```diff
@@ -57,7 +56,9 @@ class Requester:
     def __authenticate(
         self, url: str, responseHeader: Dict[str, Any], parameters: Dict[str, Any],
     ) -> None: ...
-    def __customConnection(self, url: str,) -> Optional[ReplayingHttpsConnection]: ...
+    def __customConnection(
+        self, url: str,
+    ) -> Optional[Union[HTTPRequestsConnectionClass, HTTPSRequestsConnectionClass]]: ...
```

Comment:
Both of those classes are a subclass of ReplayingConnection, does it also work if you specify that instead of the union of HTTP{,S}?